origami.json

```diff
@@ -5,11 +5,24 @@
     "support": "next.team@ft.com",
     "supportStatus": "experimental",
     "runServer": "true",
+    "demosDefaults": {
+        "sass": "demos/src/demo.scss",
+        "js": "demos/src/demo.js",
+        "documentClasses": "",
+        "dependencies": "o-fonts@^2.0.0,o-grid@^4.0.0"
+    },
     "demos": [
         {
-            "path": "/demos/light-signup-demo.html",
-            "expanded": true,
-            "description": ""
+            "name": "promo-image",
+            "description": "Light signup form with promo image",
+            "template": "demos/src/standard.html",
+            "path": "/demos/promo-image.html"
```

Comment:
I'm by no means an expert on component stuff, and we're a little thin on the ground this week on people who are! However yeah we don't want anyone to have to manually ignore changes – that's a recipe for accidental committing. 

I'd say keep the `path` property to avoid that.